### PR TITLE
Created a new webpage that generates a QR code that links to a URL.

### PR DESCRIPTION
**Pull request by lelekaz**
Also has hardcoded switches above the generate button that toggle the different pieces of information.

### CODE REVIEW

**Comment by sderacy** [comment]
Nice job taking point on this! I just ran it on my computer and it looks pretty good to start

File: views/account/generate_qr_code/index.ejs
Comment (sderacy) [inline views/account/generate_qr_code/index.ejs:50]: I think we should fine-tune the number of options we give them for the QR code dimensions. I don't think the users are necessarily gonna use all of them

**Review by sderacy — COMMENTED** [review]
Check out the comments I left. I also couldn't comment on the images but rare they just for showing how the QR code generation works?

If so, I'd like it if you placed it in public if you don't care if other people can see it or docs so that it can be considered documentation. Great work again!

**Comment by sderacy** [comment]
Also just so that I know for later, this happens when you choose 700x700:
image

That's actually my fault lol, so I will need to fix that before or after this PR gets merged. I'm currently still working on the footer since I took a break but I'll get back to you on that.

Nice work so far!

File: views/account/generate_qr_code/index.ejs
Comment (lelekaz) [reply views/account/generate_qr_code/index.ejs:50]: Sounds good! Yeah we'll discuss the number of options when we all meet.

File: views/account/generate_qr_code/index.ejs
Comment (lelekaz) [reply views/account/generate_qr_code/index.ejs:33]: I will try and edit it to be yellow, like me!
It's a bootstrap switch, so I'll try to override the color to yellow to match the theme.

File: views/account/generate_qr_code/index.ejs
Comment (paytonshaltis) [reply views/account/generate_qr_code/index.ejs]: It may even be a good idea to add this in the settings of the app: I think the size of stuff is probably a good addition to the accessibility settings.

File: views/account/generate_qr_code/index.ejs
Comment (paytonshaltis) [reply views/account/generate_qr_code/index.ejs:47]: > Sorry but when I was saying we should limit the options, I was thinking that it might still be good to have one of the bigger options like 700x700 too. This wouldn't stop me from approving the PR though so if you decide not to change it that's fine. Feel free to chime in here @paytonshaltis @thadakn1

I think we would just need to test on mobile to make sure that it all fits on the screen, but we can easily change these values later on if we need or after user testing starts.

File: views/account/generate_qr_code/main.js
Comment (paytonshaltis) [inline views/account/generate_qr_code/main.js:14]: Thanks for changing to Google Leah. Better safe than sorry I think!

Comment (paytonshaltis) [reply views/account/generate_qr_code/main.js:14]: And obviously, this function will need to change as we get further into the project. The user won't have an option to manually enter a link, obviously. Some of this code will likely need to be in an API endpoint on our server to process what accounts they want, and the server can reply with the correct link. But this is good in the client script file for showing her in our code review next week.

**Review by paytonshaltis — APPROVED** [review]
I mostly focused on the JS; @sderacy or @thadakn1 can take a closer look at the frontend stuff, but the JS looks good. Whenever you get the OK from someone on the frontend to merge go for it.

**Comment by sderacy** [comment]
Can't approve it on phone but I approve it too

File: views/account/generate_qr_code/index.ejs
Comment (lelekaz) [reply views/account/generate_qr_code/index.ejs:47]: Yeah the reason why I maxed out at 300x300 is because it goes off the screen in mobile. And also, for some reason I can't go past like 500x500 with the google carts api. But yeah we'll talk about it in person about it more.

File: views/account/generate_qr_code/style.css
Comment (lelekaz) [reply views/account/generate_qr_code/style.css:89]: Lol I don't know what it is, but if I comment it out then the toggle part turns blue 😂

File: views/account/generate_qr_code/main.js
Comment (lelekaz) [reply views/account/generate_qr_code/main.js:14]: Yeah no worries princess; always gotta keep you happy 😂 
Yes I agree, it will definitely change over the course of the project. I just had some of the stuff like inserting a manual link on the page as like a placeholder before we actually implement a link on our side.

**Comment by sderacy** [comment]
Actually before we merge this, I do have one question @paytonshaltis @lelekaz 

Right now the page allows you to enter a url to generate the QR code. Is the plan to have the user enter a url?

**Comment by paytonshaltis** [comment]
> Actually before we merge this, I do have one question @paytonshaltis @lelekaz
> Right now the page allows you to enter a url to generate the QR code. Is the plan to have the user enter a url?

Yeah, sorry this got lost in all of the comments of the PR, but here's what I said to Leah above:

> And obviously, this function will need to change as we get further into the project. The user won't have an option to manually enter a link, obviously. Some of this code will likely need to be in an API endpoint on our server to process what accounts they want, and the server can reply with the correct link. But this is good in the client script file for showing her in our code review next week.

**Review by sderacy — APPROVED** [review]

**Pull request merged**
